lib/shopify_app/controller_concerns/frame_ancestors.rb

```diff
@@ -8,7 +8,7 @@ module FrameAncestors
       content_security_policy do |policy|
         policy.frame_ancestors(-> do
           domain_host = current_shopify_domain || "*.#{::ShopifyApp.configuration.myshopify_domain}"
-          "#{ShopifyAPI::Context.host_scheme}://#{domain_host} https://admin.shopify.com"
+          "#{ShopifyAPI::Context.host_scheme}://#{domain_host} https://admin.#{::ShopifyApp.configuration.unified_admin_domain}"
```

Comment:
Also, should we add a test to `test/controllers/concerns/embedded_app_test.rb` to ensure that the ancestors are being set correctly?

Reply:
I've added some tests for the content security policy header.